packages/manager/core/application/src/hooks/pci/useProject.ts

```diff
@@ -1,5 +1,5 @@
 import { QueryOptions, useQuery } from '@tanstack/react-query';
```

Comment:
PCI related hooks and types where added in manager-components in a previous PR by @anooparveti  :
https://github.com/ovh/manager/pull/11438

You should probably rebase your branches